### ISSUE TITLE
Add possibility to inspect not whole application or stage, but concrete Nodes

### ISSUE THREAD

**Issue by PavelTurk** [mkpaz/devtoolsfx#1]
Let's consider, that I have a tab application with multiple tabs and I want to inspect only concrete tab. That's why I suggest possibility to inspect concrete Nodes, Tabs etc but not whole application

For example, my application:
```
Stage
 |- TabPane
     |-Tab 1
     |-Tab 2
        |- VBox
             |-ToolBar
             | ....
```
So, to inspect only Tab 2 (and all its children) I do:
```
var toolPane = GUI.createToolPane(tab2, Preferences preferences, tab2.getText());
```

**Comment by mkpaz** [comment]
I started with this, but as the API evolved, I abandoned it due to the high cost for a very niche feature. For example, the application listens for scene root changes and updates the UI accordingly. But what if someone replaces the Tab object, removes it from the scene graph, or moves the scene to another stage? How should the application react? These are just edge cases, and there are more. I believe it's better to focus on the root node, as it can always be obtained from any scene.
